docs-ui/stories/views/sidebarSection.stories.js

```diff
@@ -20,14 +20,13 @@ export default {
 };
 
 export const Default = ({title, icon}) => (
-  <SidebarSection
-    title={title}
-    icon={
-      icon && <QuestionTooltip position="top" title="Tooltip description" size="sm" />
-    }
-  >
-    {'16 hours'}
-  </SidebarSection>
+  <SidebarSection.Wrap>
+    <SidebarSection.Title>
+      {title}
+      {icon && <QuestionTooltip position="top" title="Tooltip description" size="sm" />}
```

Comment:
We should think about making a global IconWrapper component because this is very inconsistent everywhere too.